website/docs/r/lb_nat_pool.html.markdown

```diff
@@ -57,13 +57,13 @@ resource "azurerm_lb_nat_pool" "example" {
 The following arguments are supported:
 
 * `name` - (Required) Specifies the name of the NAT pool.
-* `resource_group_name` - (Required) The name of the resource group in which to create the resource.
 * `loadbalancer_id` - (Required) The ID of the Load Balancer in which to create the NAT pool.
 * `frontend_ip_configuration_name` - (Required) The name of the frontend IP configuration exposing this rule.
 * `protocol` - (Required) The transport protocol for the external endpoint. Possible values are `Udp` or `Tcp`.
 * `frontend_port_start` - (Required) The first port number in the range of external ports that will be used to provide Inbound Nat to NICs associated with this Load Balancer. Possible values range between 1 and 65534, inclusive.
 * `frontend_port_end` - (Required) The last port number in the range of external ports that will be used to provide Inbound Nat to NICs associated with this Load Balancer. Possible values range between 1 and 65534, inclusive.
 * `backend_port` - (Required) The port used for the internal endpoint. Possible values range between 1 and 65535, inclusive.
+* `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource.
```

Comment:
which means this is no longer forcenew

Reply:
Here is using [SchemaResourceGroupNameDeprecated() ](https://github.com/terraform-providers/terraform-provider-azurerm/blob/326ae0569f803b4fbaee7d3a40aaa56ee6568ce8/azurerm/helpers/azure/resource_group.go#L21)which doesn't include forcenew.